### ISSUE TITLE
Finetune on my own dataset

### ISSUE THREAD

**Issue by Solidays-Hung** [KAIST-VICLab/FMA-Net#21]
Thanks for your work!
I load your pretrained R&D ckpt and finetune the model on my dataset. But after 40 epochs, the model seems to collapse. The Recon Loss and Restoration Loss **suddenly increase**. Besides, the result_dict['recon'] seems to be **All black images**. Could you tell me the possible cause of this?

**Comment by GeunhyukYouk** [comment]
Hi, sorry for the late reply.

This issue is probably caused by the learning rate being too high. It might help to tune the hyperparameters of the optimizers.
